archivers/unshield/Portfile

```diff
@@ -12,25 +17,18 @@ long_description \
 	Extract files from InstallShield-style cabinet files (usually named \
 	data1.cab, data1.hdr, data2.cab, etc), versions 5 and later
 
+depends_lib			port:libiconv \
+                    port:zlib
```

Comment:
the resulting binary and dylib link with `openssl`:
```
% otool -L /opt/local/lib/libunshield.dylib
/opt/local/lib/libunshield.dylib:
	/opt/local/lib/libunshield.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/opt/local/lib/libiconv.2.dylib (compatibility version 9.0.0, current version 9.1.0)
	/opt/local/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/opt/local/libexec/openssl11/lib/libssl.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/opt/local/libexec/openssl11/lib/libcrypto.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1281.100.1)
```

it seems there is an option to build without it as there is a CMake flag for it. So to get "reproducible builds" you either need to declare a dependency on it or make sure that it will not be used even if a user has it installed.

Reply:
Added the dependency!